frontend/src/contexts/AuthContext.js

```diff
@@ -65,10 +74,26 @@ export const AuthProvider = ({ children }) => {
             const searchWithEmail = whitelist.includes(userEmail)
             try {
                 const identifier = searchWithEmail ? userEmail : userWatiam
```

Comment:
lets standardize on just only email tbh. I know this was already there but now that I think of it we should stick with email either way, even if its a white list search

Reply:
what this would entail would be to remove any sort of watIam logic

Reply:
resolved in call. turns out we do need this